src/reanimated2/js-reanimated/index.ts

```diff
@@ -27,7 +27,7 @@ const reanimatedJS = new JSReanimated();
 global._makeShareableClone = (c) => c;
 global._scheduleOnJS = (func, args) => {
   if (args) {
-    queueMicrotask(() => func(...args));
+    queueMicrotask(() => func(args));
```

Comment:
Are you sure that we can remove spread operator here?

Reply:
No, this is a mistake.